truckmanagement-transportation-microservice/src/main/java/com/transportation/microservice/Model/Route.java

```diff
@@ -0,0 +1,46 @@
+package com.transportation.microservice.Model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonTypeId;
+import lombok.*;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+import org.springframework.web.bind.annotation.CrossOrigin;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+import java.time.LocalDate;
+
+
+@Getter
+@Setter
```

Comment:
lombok fails to create a setter on my copy, but was fixed with manually creating the setRouteId() method